src/server/Html.js

```diff
@@ -27,7 +27,12 @@ export default function Html({store, entries, StyleSheetServer, renderProps}) {
     );
   });
   const dehydratedStyles = `window.__APHRODITE__ = ${JSON.stringify(css.renderedClassNames)}`;
-  const auth0ClientOptions = `window.__AUTH0__ = ${JSON.stringify(auth0)}`;
+  const clientOptions = `
+    window.__ACTION__ = {
+      auth0: ${JSON.stringify(auth0)},
+      sentry: ${JSON.stringify(process.env.SENTRY_DSN_PUBLIC)}
```

Comment:
That's right, we'll have two separate DSNs. One for staging and one for production.